src/app-directory/specification/appd.yaml

```diff
@@ -347,9 +347,15 @@ components:
     Icon:
       description: Icon holder
       properties:
-        icon:
+        src:
```

Comment:
This is a breaking change. Considering that this is meant to be part of the "2.0" release I thought that it would be  I acceptable. 
If not, I suggest to either 
 - not introduce `src`  and use the `icon` property as image url
 - deprecate icon in favour of a newly added `src` property. In that case Desktop Agent may treat `icon` as it would treat `src`